packages/presentation/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
 
   flutter_bloc: ^8.1.1
   equatable: ^2.0.5
+  get_it: ^7.2.0
```

Comment:
package "get_it" should only be in the domain layer

Reply:
ok, corrected